tests/src/Context/JoinupTrait.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Drupal\joinup\Context;
+
+
+use Drupal\Component\Uuid\Php;
+use Drupal\file\Entity\File;
+use Drupal\rdf_entity\Entity\Rdf;
+
+/**
+ * Helper trait for behat tests.
+ */
+trait JoinupTrait {
```

Comment:
This entire file should be removed. This is originating from ISAICP-2401 but it has been removed before that ticket was finished. All this code is now split up in separate traits such as RdfEntityTrait etc.